examples/morpheusvm/tests/transfer.go

```diff
@@ -24,23 +23,31 @@ import (
 // ref https://onsi.github.io/ginkgo/#mental-model-how-ginkgo-traverses-the-spec-hierarchy
 var TestsRegistry = &registry.Registry{}
 
-var _ = registry.Register(TestsRegistry, "Transfer Transaction", func(t ginkgo.FullGinkgoTInterface, tn tworkload.TestNetwork) {
-	require := require.New(t)
-	other, err := ed25519.GeneratePrivateKey()
-	require.NoError(err)
-	toAddress := auth.NewED25519Address(other.PublicKey())
-
-	authFactory := tn.Configuration().AuthFactories()[0]
-	tx, err := tn.GenerateTx(context.Background(), []chain.Action{&actions.Transfer{
-		To:    toAddress,
-		Value: 1,
-	}},
-		authFactory,
-	)
-	require.NoError(err)
-
-	timeoutCtx, timeoutCtxFnc := context.WithDeadline(context.Background(), time.Now().Add(30*time.Second))
-	defer timeoutCtxFnc()
-
-	require.NoError(tn.ConfirmTxs(timeoutCtx, []*chain.Transaction{tx}))
-})
+var _ = registry.Register(TestsRegistry, "Transfer Transaction",
+	func(t ginkgo.FullGinkgoTInterface, tn tworkload.TestNetwork, authFactories ...chain.AuthFactory) {
+		require := require.New(t)
+		ctx := context.Background()
+		targetFactory := authFactories[0]
+		authFactory := tn.Configuration().AuthFactories()[0]
+
+		client := jsonrpc.NewJSONRPCClient(tn.URIs()[0])
+		balance, err := client.GetBalance(ctx, targetFactory.Address())
+		require.NoError(err)
+		require.Equal(uint64(1000), balance)
+
+		tx, err := tn.GenerateTx(ctx, []chain.Action{&actions.Transfer{
+			To:    targetFactory.Address(),
+			Value: 1,
+		}},
```

Comment:
I think the flow should be `authFactories` are assumed to be pre-funded with a specific amount (1000) in this case, and the test flow should be:
1. confirm the provided balance is as expected
2. generate and confirm transaction _from_ the provided key (rather than the auth factory from the configuration)
3. confirm balance updates after confirming the tx

Reply:
I have changed the test to have a pre-funded authFactory sending to another pre-funded authFactory 👍 
I thought it was interesting to use _both_ `pre-funded authFactory` and `config authFactory`. So the users see the two possibilities.